src/components/Checkbox/Checkbox.story.js

```diff
@@ -67,4 +68,40 @@ storiesOf('Checkbox', module)
         Disabled
       </Checkbox>
     ))
+  )
+  .add(
+    'Multiple Checkboxes',
+    withInfo()(() => {
+      const initialValues = { apples: false, bananas: false, oranges: false };
+      const withCheckboxes = withState('values', 'onChange', initialValues);
```

Comment:
Somebody did not want to use our `State` component? 🧐

Reply:
I still find that this is just adding more lines of code and is less elegant compared to this one line. Sowwy!

Reply:
Actually btw, I think I tried using the `State` component btw and I got infinite loops when working with objects rather than primitive values. Is your state component using withState under the hood or a custom implementation?